spec/features/adventures/user_creates_new_adventure_with_location_spec.rb

```diff
@@ -10,20 +10,20 @@
 #    google maps
 # [] The serach box will autocomplete as I type
 # [] The search box will show the adventure location on a map
-# [] I recieve a success message when I successfully add an adventure
+# [x] I recieve a success message when I successfully add an adventure
+# [x] If I do not sucessfully save an adventure with location only, I am told why
```

Comment:
Line is too long. [81/80]<br>Incorrect indentation detected (column 0 instead of 2).